lib/install/validate/validator.go

```diff
@@ -156,6 +157,30 @@ func NewValidator(ctx context.Context, input *data.Data) (*Validator, error) {
 	return v, nil
 }
 
+var schemeMatch = regexp.MustCompile(`^\w+://`)
```

Comment:
How to in this case, since it could be "http", "https", maybe even another scheme.